dealroom_firestore_connector/__init__.py

```diff
@@ -156,6 +157,42 @@ def collection_exists(collection_ref: CollectionReference):
     return len(docs) > 0
 
 
+from google.cloud.firestore_v1.document import DocumentReference
+
+
+def get_history_doc_refs(
+    db: firestore.Client,
+    finalurl_or_dealroomid: str
+) -> Tuple[DocumentReference]:
+    """Returns a DocumentReference based on the final_url field or dealroom_id
+    field.
+
+    Args:
+
+    Returns:
+
+    Examples:
+        >>> db = new_connection(project=FIRESTORE_PROJECT_ID)
+        >>> doc_refs = get_history_refs(db, "dealroom.co")
+    """
+
+    collection_path = "history"
+    collection_ref = db.collection(collection_path)
+
+    if finalurl_or_dealroomid.isnumeric():
+        query_params = ["dealroom_id", "==", int(finalurl_or_dealroomid)]
+    else:
+        query_params = ["final_url", "==", finalurl_or_dealroomid]
+
+    query = collection_ref.where(*query_params)
+    docs = stream(query)
+    if docs == -1:
+        logging.error("Couldn't stream query.")
+        return False
```

Comment:
This is wrong, sorry. What should I return in case of error? the same ERROR code that was raised?